src/utils/helperFunctions.ts

```diff
@@ -0,0 +1,18 @@
+import isMatch from 'lodash/isMatch'
+import pick from 'lodash/pick'
+
+import { IUserResponse } from '../models/IUserResponse'
+import { IEditUserProfileForumParams } from '../models/IUser'
+
+/**
+ * Функция сравнивает два объекта по ключам
+ */
+const checkUserPropsAreUnequal = ({ objA, objB, keys }: {
```

Comment:
❗
`Unequal `провоцирует двойное отрицание, это сложно парсится. Плюс функция может проверять что угодно, не только юзеров
```suggestion
const checkObjectKeysAreEqual = ({ objA, objB, keys }: {
```

Reply:
Готово!